src/server/github_handler.rs

```diff
@@ -372,8 +368,15 @@ impl GithubEventHandler {
                 notify_channel_only = true;
             }
 
+            let commits: Vec<github::Commit>;
+            if verb.is_some() || self.action == "labeled" {
```

Comment:
duplicating this logic from below is sorta sad. this was mostly to avoid looking up things for entries we totally don't intend to process. 

can probably make this clearer.